File: Android/TrainigTaskAndroid/com.qulix.panteleevrv.trainingtask/src/main/java/com/qulix/panteleevrv/trainingtask/server/ServerException.java
package com.qulix.panteleevrv.trainingtask.server;

/**
 * Класс серверной ошибки.
 *
 * @author Q-RPA
 */
public class ServerException extends Exception {

    /**
     * Конструктор класса {@link ServerException}
     */
    public ServerException() {
    }

    /**
     * Конструктор класса {@link ServerException}
     *
     * @param message сообщение об ошибке
     */
    public ServerException(String message) {
        super(message);
    }

    /**
     * Конструктор класса {@link ServerException}
     *
     * @param message сообщение об ошибке
     * @param throwable причина вызова исключения
     */
    public ServerException(String message, Throwable throwable) {
        super(message, throwable);
    }
}

File: WP/Qulix.PanteleevRV.TrainingTask.WpClient/TasksWindowsPhone/Server/ServerException.cs
using System;

namespace TasksWindowsPhone.Server
{
    /// <summary>
    /// Класс серверной ошибки.
    /// </summary>
    public class ServerException : Exception
    {
        /// <summary>
        ///     Конструктор класса
        /// </summary>
        /// <param name="message">Текст сообщения ошибки</param>
        public ServerException(string message) : base(message)
        {
        }
    }
}
